scripts/actions/utils/serialization-helpers.js

```diff
@@ -14,6 +14,13 @@ const removeParagraphs = () => (tree) => {
   });
 };
 
+const createJsonStr = (str) =>
```

Comment:
could you add a doc comment explaining what this does/is used for? i think it turns a string from something like `{ something: 'hey' }` into `{ "something": "hey" }`? but it'd be good to be clear about it and why it's needed. is it because MDX doesn't automatically recognize JS objects in props?

Reply:
ah yeah, the serializer has access to the props as a string so "{key: 'value'}" but to convert it to an object via JSON.parse it needs to be a json string